core/lib/resources/presence/presence_manager.js

```diff
@@ -39,13 +40,30 @@ PresenceManager.prototype.setup = function() {
   });
 
   // save so you removeListener on destroy
-  this.sentryListener = function(sentry) {
+  this.sentryListener = function (sentry) {
+    if (manager.destroying) return;
+
     var clientIds = store.clientsForSentry(sentry);
 
     var chain = function (clientIds) {
+      var immediateObject;
       var clientId = clientIds.pop();
 
-      if (!clientId) return;
+      // Pick up any new users/clients that have subscribed to this resource
+      if (!clientId && !manager.destroying) {
+        clientIds = store.clientsForSentry(sentry);
```

Comment:
Correct.  clientsForSentry() picks up the resource clients for a given sentry, which in this case is assumed to be dead and so the list will not change.  I was confusing this list with the local list of clients on the current server for the same resource.